tests/Unit/Controllers/Metrics/StatsTest.php

```diff
@@ -431,9 +431,9 @@ public function testCurrentlyWorkingUsers(): void
         /** @var Shift $shift */
         $shift = Shift::factory()->create(['start' => Carbon::now()->subHour(), 'end' => Carbon::now()->addHour()]);
 
-        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded' => false]);
-        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded' => false]);
-        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded' => true]);
+        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded_by' => null]);
+        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded_by' => null]);
+        ShiftEntry::factory()->create(['shift_id' => $shift->id, 'freeloaded_by' => 42]);
```

Comment:
This needs a valid user instead of just an ID

Reply:
TELL ME HOW??????????